constants.py

```diff
@@ -98,6 +98,13 @@ def error_string(error, logname, with_hd):
 logsfs_label_prefix = 'logs-'
 logsfs_label = logsfs_label_prefix + '%s'
 
+rootpart_label = "root"
+backuppart_label = "backup"
+lvmpart_label = "Local SR"
+bootpart_label = "EPT Boot"
+logspart_label = "logs"
+swappart_label = "swap"
```

Comment:
These want to be consistently capitalised or not, but as they're names, I'd suggest with a capital letter.

Reply:
IMHO lower case would be consistent with filesystem labels we already use